test/tinycore.txt

```diff
@@ -2,8 +2,15 @@ These are binaries from
     http://tinycorelinux.net
 with the following patch applied:
 
+Source is available http://www.tinycorelinux.net/6.x/x86/release/src/
+
+curl -O http://www.tinycorelinux.net/6.x/x86/release/distribution_files/vmlinuz64
+# c4f46b5cd826d7a765a3e85439309cfc  vmlinuz64
+mv vmlinuz64 vmlinuz
+curl -O http://www.tinycorelinux.net/6.x/x86/release/distribution_files/core.gz
+mv core.gz initrd.gz
 mkdir initrd
-( cd initrd ; zcat ../initrd.gz | sudo cpio -idm )
-sudo sed -i '/^# ttyS0$/s#^..##' initrd/etc/securetty 
-sudo sed -i '/^tty1:/s#tty1#ttyS0#g' initrd/etc/inittab
+( cd initrd ; gzcat ../initrd.gz | sudo cpio -idm )
+sudo sed -i -e '/^# ttyS0$/s#^..##' initrd/etc/securetty 
+sudo sed -i -e '/^tty1:/s#tty1#ttyS0#g' initrd/etc/inittab
 ( cd initrd ; find | sudo cpio -o -H newc ) | gzip -c > initrd.gz
```

Comment:
Did this final `find` (in the context not the modified code) really work for you on OSX? I think it needs to be `find .`. What I see locally is:
```
$ ( cd initrd ; find | sudo cpio -o -H newc ) | gzip -c > initrd.gz
usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
       find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
1 block
```

Perhaps it would be best to pull this stuff out into an actual shell script rather than embedding it in a txt file?  (actually, really a renaming in practice I think since there isn't much else in the txt)

Lastly if we are going to include a checksum IMHO it should be something more secure than md5, like  `shasum` (AKA sha1) or `shasum -s 256` and we should include it for both the kernel and the initrd. But it seems like upstream only publishes the md5 (and unsigned), so probably we should just omit it rather than providing a false sense of security.